### PR TITLE
make functions and fields private/protected if needed

### PR DESCRIPTION
**Pull request by jchen293**
# Description

make functions and fields private/protected

# Testing

Unit tests passed

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: src/services/address_service.js [hunk 1]
```diff
@@ -2,19 +2,19 @@ import baseService from './base_service';
 
 export default (easypostClient) =>
   class AddressService extends baseService(easypostClient) {
-    static _name = 'Address';
+    static #_name = 'Address';
```
Comment (Justintime50) [inline src/services/address_service.js:5]: We don't want these to be both pounded (#) and underscored. They should only be one or the other I'd think

**Review by Justintime50 — CHANGES_REQUESTED** [review]
It looks like there's a bit of a mix of usage here. We can comb through this next week an make sure we catch all the items needed

File: src/services/address_service.js (hunk 1 above)
Comment (jchen293) [reply src/services/address_service.js:5]: Oops, this is a typo here, but for the private fields it should be using `#`

**Review by nwithan8 — DISMISSED** [review]
LGTM as long as you address what Justin brought up.

File: src/services/carrier_account_service.js [hunk 2]
```diff
@@ -71,20 +71,20 @@ export default (easypostClient) =>
...
+    static _selectCarrierAccountCreationEndpoint(carrierAccountType) {
       if (Constants.CARRIER_ACCOUNTS_WITH_CUSTOM_WORKFLOWS.includes(carrierAccountType)) {
         return 'carrier_accounts/register';
       }
       return 'carrier_accounts';
     }
 
     /**
-     * Retrieve a list of all carrier accounts associated with the API key.
+     * Retrieve a list of all carrier accounts associated with the API #.
```
Comment (Justintime50) [inline src/services/carrier_account_service.js:82]:
```suggestion
     * Retrieve a list of all carrier accounts associated with the API key.
```

File: src/services/end_shipper_service.js [hunk 3]
```diff
@@ -44,18 +44,18 @@ export default (easypostClient) =>
...
     static async retrieve(id) {
-      const url = `${this._url}/${id}`;
+      const url = `${this.#url}/${id}`;
 
       return this._retrieve(url);
     }
 
     /**
-     * Retrieve a list of all EndShippers associated with the API key.
+     * Retrieve a list of all EndShippers associated with the API #.
```
Comment (Justintime50) [inline src/services/end_shipper_service.js:53]:
```suggestion
     * Retrieve a list of all EndShippers associated with the API key.
```

File: src/services/shipment_service.js [hunk 4]
```diff
@@ -209,12 +209,12 @@ export default (easypostClient) =>
     }
 
     /**
-     * Retrieve a list of all shipments associated with the API key.
+     * Retrieve a list of all shipments associated with the API #.
```
Comment (Justintime50) [inline src/services/shipment_service.js:212]:
```suggestion
     * Retrieve a list of all shipments associated with the API key.
```

**Review by Justintime50 — CHANGES_REQUESTED** [review]
Nice, properties use pound, functions use underscore.

There were a couple instances where a docstring got clobbered with find and replace that needs fixing.

File: src/services/carrier_account_service.js (hunk 2 above)
Comment (jchen293) [reply src/services/carrier_account_service.js]: Ooo, nice catch! This was unintended, search and replace gone wrong 😅

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**